src/Api/Resolver/FacetsResolver.php

```diff
@@ -47,6 +47,8 @@ public function resolve(array $data): array
             return [];
         }
 
-        return $adapter->getAggregations();
+        return array_filter($adapter->getAggregations(), function ($facet) {
+            return [] !== $facet['buckets'] ?? [];
```

Comment:
This condition seems wrong. It rather should be `[] !== ($facet['buckets'] ?? [])`. I'm facing the issue related to this, I guess:
```
Warning: Undefined array key \u0022buckets\u0022 in \/srv\/sylius\/vendor\/bitbag\/elasticsearch-plugin\/src\/Api\/Resolver\/FacetsResolver.php line 51
```